docs/src/developers_guide/contributing_ci_tests.rst

```diff
@@ -121,3 +120,5 @@ See the `pre-commit.ci dashboard`_ for details of recent past and active Iris jo
 
 .. _.pre-commit-config.yaml: https://github.com/SciTools/iris/blob/main/.pre-commit-config.yaml
 .. _pre-commit.ci dashboard: https://results.pre-commit.ci/repo/github/5312648
+.. _CLA Assistant: https://github.com/cla-assistant/cla-assistant
+.. |SciTools Contributor's License Agreement (CLA)| replace:: **SciTools Contributor's License Agreement (CLA)**
```

Comment:
https://stackoverflow.com/a/63394243

Reply:
This is cool! I've had this issue before (though I imagine I'll forget this before I need it next 😂)